vosekast_control/Vosekast.py

```diff
@@ -307,6 +297,19 @@ def clean(self):
 
         self.logger.debug('Vosekast stopped.')
 
+    def state_overview(self):
+        # for tank in self.tanks:
+        #     MQTTConnection.publish_message(StatusMessage('tank', tank.name, tank.state))
+        # for pump in self.pumps:
+        #     MQTTConnection.publish_message(StatusMessage('pump', pump.name, pump.state))
+        # for valve in self.valves:
+        #     MQTTConnection.publish_message(StatusMessage('valve', valve.name, valve.state))
+        # for level_sensor in self.level_sensors:
+        #     MQTTConnection.publish_message(StatusMessage('level_sensor', level_sensor.name, level_sensor.state))
```

Comment:
this old code can be removed. He no longer has any reason to be.

Reply:
ok